prow/plugins/bugzilla/bugzilla.go

```diff
@@ -335,6 +353,16 @@ Please contact an administrator to resolve this issue, then request a bug refres
 				}
 				response += fmt.Sprintf(" The bug has been moved to the %s state.", *options.StatusAfterValidation)
 			}
+			if options.AddExternalLink != nil && *options.AddExternalLink {
```

Comment:
We can fetch the current state of external tracked bugs and do this conditionally as well. Could be a follow-up or in this one.

Reply:
I would save a web request and just let Bugzilla handle it. From the docs linked, it looks like it even sends back what was actually changed.  That's something we could add to the bot's comment.